content/guides/hosting/hosting-options/self-managed/ref-arch.md

```diff
@@ -91,7 +89,6 @@ W&B runs on the Intel (x86) CPU architecture. ARM is not supported.
 Terraform is the recommended way to deploy W&B for production. Using Terraform, you define the required resources, their references to other resources, and their dependencies. W&B provides Terraform modules for the major cloud providers. For details, refer to [Deploy W&B Server within self managed cloud accounts]({{< relref "/guides/hosting/hosting-options/self-managed.md#deploy-wb-server-within-self-managed-cloud-accounts" >}}).
 
 ## Sizing
-
```

Comment:
FYI, this doesn't really matter for output, and I wouldn't worry about it next time. :)